### PR TITLE
Reach Router

### PR DESCRIPTION
**Pull request by kylewhitaker**
👋 @LGG233 
🌵 [Trello card](https://trello.com/c/2ioK38Yg)
🤠 Added @reach/router
🖕 Removed react-router-dom
 😜 I think you're gonna like this better...

### CODE REVIEW

**Comment by LGG233** [comment]
> 👋 @LGG233
> 🌵 [Trello card](https://trello.com/c/2ioK38Yg)
> 🤠 Added @reach/router
> 🖕 Removed react-router-dom
> 😜 I think you're gonna like this better...

Thanks; this looks good. Also: I see you cleaned up the import statements in app.js - didn't know you could import that way. Thanks!

**Comment by kylewhitaker** [comment]
[add Reach Router to React app](https://trello.com/c/2ioK38Yg/9-add-reach-router-to-react-app)

File: client/src/App.js [hunk 1]
```diff
@@ -61,30 +56,15 @@ class App extends Component {
...
-                    <SignIn
-                      updateUser={this.updateUser}
-                    />}
-                />
-              </Switch>
+              <Router>
+                <Landing path="/" />
+                <SignUp path="/signup" signup={this.signup} />
+                <SignIn path="/signin" updateUser={this.updateUser} />
+              </Router>
```
Comment (kylewhitaker) [inline client/src/App.js:63]: @stuartt this is a test of a public repo comment

Comment (Stuartt) [reply client/src/App.js:63]: Got it.

**Pull request merged**